stacktest/fakemetrics/out/kafkamdm/kafkamdm.go

```diff
@@ -83,6 +88,8 @@ func New(topic string, brokers []string, codec string, stats met.Backend, partit
 		part, err = p.NewKafka("byOrg")
 	case "bySeries":
 		part, err = p.NewKafka("bySeries")
+	case "bySeriesWithTags":
+		part, err = p.NewKafka("bySeriesWithTags")
 	case "lastNum":
 		lmPart = LastNumPartitioner{}
```

Comment:
LastPartitioner implements the p.Partitioner interface, so we can just assign it to `part`  (need to also change part to be `p.Partitioner` instead of `*p.Kafka`)

Reply:
You also need to remove the `config.Producer.Partitioner = sarama.NewManualPartitioner` below, as we now always use the NewManualPartitioner